packages/store/src/IStoreDynamicPartial.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { IErrors } from "./IErrors.sol";
+
+interface IStoreDynamicPartial is IErrors {
```

Comment:
is this ~always going to be used for arrays? just wondering if we should name it more by what it's designed to be used for rather than how it's implemented